README.md

```diff
@@ -224,7 +224,7 @@ Required Rust version `>=1.65`.
 
 ## Contributing
 
-Want to contribute? Read our [contribution guideline](./CONTRIBUTING.md).
+Want to contribute? Read our [contribution guideline](../CONTRIBUTING.md).
```

Comment:
Not sure where this drive-by change comes from, but it should be reverted.

Reply:
Well, I was trying to open the document, but this markdown file lives in py-polars. Where there is no CONTRIBUTING.md file present. But there is one in the root (.. instead of .).

Should I make a separate PR for this small change?

Reply:
> this markdown file lives in py-polars

It does not - it lives in the root :)

Reply:
Yes, you're absolutely right. I didn't see that because in my source tree (windows...), this file is a real file. Not a link.
However, something worth considering:

- browse to https://github.com/pola-rs/polars/tree/main/py-polars (search for Contributing)
- click the link
- ... 404 ...

Minor issue, but I thought to fix it. Well, I would have broken it for just about everyone else ;-)

Reply:
Hm yea that is a small issue. Let's keep that out of this PR though! Feel free to submit a separate fix for that if you find a nice solution. Maybe we should just get rid of that symlinked README in `py-polars`, but we need to make sure the README does get packaged together with the wheels.

Reply:
Maybe just symlink CONTRIBUTING.md too?